backend/src/main/resources/logback.xml

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<configuration>
+    <!-- Stop output INFO at start -->
+    <statusListener class="ch.qos.logback.core.status.NopStatusListener" />
+
+    <appender name="Console" class="ch.qos.logback.core.ConsoleAppender">
+        <layout class="ch.qos.logback.classic.PatternLayout">
+            <Pattern>
+                %black(%d{ISO8601}) %highlight(%-5level) [%blue(%t)] %yellow(%C{1.}): %msg%n%throwable
+            </Pattern>
+        </layout>
+    </appender>
+    <appender name="CConsole" class="ch.qos.logback.core.ConsoleAppender">
+        <encoder class="ch.qos.logback.classic.encoder.PatternLayoutEncoder">
+            <pattern>%green(%d{ISO8601}) %highlight(%-5level) [%blue(%15.15t)] %yellow(%40.40C): %msg%n%throwable</pattern>
+        </encoder>
+    </appender>
+
+    <logger name="com.flow" level="debug"/>
+    <logger name="ch.qos.logback" level="warn"/>
+    <logger name="org.hibernate.SQL" level="debug" />
```

Comment:
Szeretem, de ellenőrizd pls, hogy a JpaRepositoryval együtt kell-e, vagy elég az application.yaml megfelelő rész. Ne legyen itt fölöslegesen, legyen tiszta, hogy mi, mi miatt kerül a logba.

- [ ] Ellenőrizd, hogy az application.yaml, vagy emiatt a konfig miatt kerül be a logba az sql, a fölöslegeset töröld